superset/charts/schemas.py

```diff
@@ -1205,7 +1205,7 @@ class ImportV1ChartSchema(Schema):
     slice_name = fields.String(required=True)
     viz_type = fields.String(required=True)
     params = fields.Dict()
-    query_context = fields.Dict()
+    query_context = fields.String()
```

Comment:
This caused a schema validation exception and needed to be changed to the string type